build/gulpfile.vscode.linux.js

```diff
@@ -15,12 +15,31 @@ const util = require('./lib/util');
 const packageJson = require('../package.json');
 const product = require('../product.json');
 
+const linuxPackageRevision = Math.floor(new Date().getTime() / 1000);
+
+const flatpakManifest = {
+	appId: 'com.microsoft.CodeOSS',
+	sdk: 'org.freedesktop.Sdk',
+	runtime: 'org.freedesktop.Sdk',
+	runtimeVersion: '1.4',
+	base: 'io.atom.electron.BaseApp',
+	baseFlatpakref: 'https://s3-us-west-2.amazonaws.com/electron-flatpak.endlessm.com/electron-base-app-master.flatpakref',
+	command: "code-oss",
+	symlinks: [
+		['/share/code-oss/code-oss', '/bin/code-oss'],
+	],
+	finishArgs: [
+		'--share=ipc', '--socket=x11',  // Allow showing X11 windows.
+		'--filesystem=home',            // Allow access to home directory.
```

Comment:
I think for a text editor, exposing as much of the filesystem as possible would be a good idea. So use `--filesystem=host` instead. That's what gnome-builder does, for example.